tutorials/polkadot-sdk/parachains/zero-to-hero/execute-runtime-upgrade.md

```diff
@@ -0,0 +1,193 @@
+---
```

Comment:
This tutorial doesn't show up in the left navigation. Need to update the .pages file

Reply:
If we don't plan on having this for launch, we need to delete it because it will show up in the search bar and on search engines. Leaving it out of the .pages file isn't enough

Reply:
That tutorial is blocked until a new release of the omni node fixes an issue. I'll remove it